metron-platform/metron-indexing/src/test/java/org/apache/metron/indexing/dao/metaalert/MetaAlertIntegrationTest.java

```diff
@@ -985,6 +1024,30 @@ protected boolean findCreatedDocs(List<GetRequest> getRequests)
     throw new OriginalNotFoundException("Count not find guids after " + MAX_RETRIES + "tries");
   }
 
+  @SuppressWarnings("unchecked")
+  protected void assertEquals(Map<String, Object> expected, Map<String, Object> actual) {
```

Comment:
This method is needed to handle type mismatches (Integer to Long and Float to Double) and child alert ordering.
 These fields are not defined in the metaalert template so ES is guessing the type.  When read back some field types are different vs when they were written.